website/docs/dashboard/dashboard_user_guide.md

```diff
@@ -128,96 +123,80 @@ The settings you can set on this page are:
-- `Bootloader Mode`: Similar to `Mount CIRCUITPY`,
-  when you would like to update the UF2 file on the dashboard you will need to
-  place it in a `Bootloader` state. With this button you can perform that action.
+A purple square indicates the need to re-calibrate your pendant.
 
-:::tip
-For more information about Firmware Updates or how to perform an update,
-please visit [Dashboard Firmware Update](https://amiga.farm-ng.com/docs/dashboard/dashboard-fw)
+:::caution
+If your Amiga begins to move on its own at start up, chances are that your pendant needs to be calibrated.
+With the above image in mind (purple square around the white dot), your Amiga would move forwards
+and towards the right at start up.
 :::
 
-### H-bridge settings
+<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/8bc4980b-d6f4-445b-b772-0b7e46b7eb1a"/>
 
-<img src="https://user-images.githubusercontent.com/64480560/233127456-1f294f6f-69ac-4563-a070-9b25c2e78ac3.jpg"
-     width="320"
-     height="300" />
+Pressing the Pendant image on the right will show the latest button configurations.
+
+:::tip
+The arrows (up/down/left/right) in the center of the pendant are used to select and engage with the H-Bridge(s).
+:::
 
-This page shows the status of the H-bridge linear actuators on your Amiga.
-The H-bridge actuators are controlled with the arrow keys on the pendant.
-Only one H-bridge can be actuated at a time,
-so the `left` - `right` arrow keys are to select the active H-bridge device
-and the `up` - `down` arrows are to move an `Active` actuator.
-There are three states you will see for the H-bridge actuators.
+<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/0f880f2a-3958-4066-989d-d68b68547665"/>
 
-- Red: `Not detected`
-- Yellow: `Idle` (detected, but not selected)
-- Green: `Active` (detected and selected)
+These are the button configurations for those of you who purchased a Kartech control.
 
-### PTO settings
+### H-bridge settings
 
-Currently only 1 PTO (power take-off) device is supported.
-You will see a message that no PTO is detected if there is not a detected PTO device.
-> If you see the "No PTO" message and expect a PTO device, contact farm-ng for support by visiting
-> [our Support page](https://amiga.farm-ng.com/docs/support/).
+<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/bd24caf0-01c7-4647-b6ba-9ce6c55c47aa"/>
 
-<img src="https://user-images.githubusercontent.com/53625197/233201899-59ddb864-75dd-45d3-86c9-bdb7b3c4a358.jpg"
-     width="320" />
+An H bridge circuit allows you to apply a voltage across a load (i.e. a linear actuator)
+in either direction. We currently support up to four h bridge modules on the Amiga.
+They can be engaged momertarily or lached.
+The `up` - `down` arrows are used to `Active` the H bridge
+and the `left` - `right` arrow keys are used to select between H-bridge devices.
+As of this writing only one H-bridge device can be actuated at a time.
```

Comment:
I thought we can couple in the released fw? If not, we can say starting with `v0.5.0` (unless there is a bug that needs to be fixed?)
```suggestion
Two H-bridge devices (0 & 1 or 2 & 3) can be coupled with the **coupler** button and actuated together.
```